modules/porous_flow/paper.bib

```diff
@@ -33,13 +33,19 @@ @misc{opm
 }
 
 
-@misc{permann2019moose,
-    title={MOOSE: Enabling Massively Parallel Multiphysics Simulation},
-    author={Cody J. Permann and Derek R. Gaston and David Andrs and Robert W. Carlsen and Fande Kong and Alexander D. Lindsay and Jason M. Miller and John W. Peterson and Andrew E. Slaughter and Roy H. Stogner and Richard C. Martineau},
-    year={2019},
-    eprint={1911.04488},
-    archivePrefix={arXiv},
-    primaryClass={cs.MS}
+@article{permann2020moose,
```

Comment:
There should be no need for this reference, it is already in framework/doc/content/bib/moose.bib, which the porous flow documentation can use. I will fix this up for you.

Reply:
I think you  can ignore this, you seem to have something else going on here outside of normal docs.

Reply:
It needs to be picked up by the JOSS generator, which uses `pandoc-citeproc`. One could reference a common bib file, but there's probably no particular need.